apps/redi-connect/src/components/organisms/RediHeroLanding.scss

```diff
@@ -56,12 +56,20 @@
     @include desktop() {
```

Comment:
The same rule is repeated three times, for desktop, mobile and tablet. Why not just declare it just a single time? That should cover all these three cases

Reply:
You are right, I have updated it.